### ISSUE TITLE
Binary Ninja port

### ISSUE THREAD

**Issue by seanthegeek** [idiom/stackstack#11]
Hi @idiom,

I thought you might like to know that your plugin seems to be better at finding some stack strings than FLOSS! I watched herrcore use it to decode stack strings in a sample of Zharkbot that FLOSS could not identify. I've opened a bug report in the FLOSS project here: https://github.com/mandiant/flare-floss/issues/1027

I prefer working with Binary Ninja over IDA. I like the UI and price point much better. I was able to identify where the stack strings are but couldn't decode them.  Would you consider porting this awesome plugin to Binary Ninja? I know Python well, but I'm just getting familiar with reverse engineering platforms, so I think porting it myself is beyond my skills at this point.  A license is also missing from this project, so legally, I think you are the only one who can port it right now.

**Comment by idiom** [comment]
Yeah I'll take a look.  It could use some TLC, so looking at binja would be a fun project.
